services/virus_scan/clamav_mod.c

```diff
@@ -125,7 +125,11 @@ struct virus_db {
 #ifndef HAVE_LIBCLAMAV_095
 struct cl_limits limits;
 #endif
+#ifndef HAVE_LIBCLAMAV_0101
 unsigned int CLAMSCAN_OPTIONS = CL_SCAN_STDOPT;
+#else
+struct cl_scan_options CLAMSCAN_OPTIONS = { 0, 0, 0, 0, 0 };
+#endif
```

Comment:
It is better to leave CLAMSCAN_OPTIONS unset in the case of HAVE_LIBCLAMAV_0101

Reply:
Yes, I do think handling some of them would be good. However, not all make sense in this context.
I believe I have addressed your concerns in the current version.